### PR TITLE
ocp-rebase: use -s in cherry-pick

### PR DESCRIPTION
**Pull request by dougsland**
Helps identify the author of cherry-pick/rebase

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

File: hack/ocp-rebase.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ echo "Cherry-picking commits since ${last_merge_commit}..."
 for commit in $(git --no-pager log --oneline --reverse --no-merges ${last_merge_commit}..${downstream_remote}/${downstream_target_branch} | awk '{print $1}'); do
     echo "cherry-picking "$(git --no-pager log --format=%s -n 1 ${commit} ${downstream_remote}/${downstream_target_branch})
 
-    if ! git cherry-pick -x $commit; then
+    if ! git cherry-pick -s -x $commit; then
```
Comment (cybertron) [inline hack/ocp-rebase.sh:51]: I'm not sure we should be doing this. If a commit wasn't signed off in the first place, why would it be appropriate to add it as part of the rebase?

Comment (dougsland) [reply hack/ocp-rebase.sh:51]: Hey @cybertron,

If the commit was not signed off, it should not be merged at all, the github/openshift bot should be smart enough to not merged it as ask the author to sign. 

For the second question: As others open source projects, the person who did the rebase must sign as well to indicate who this the rebase. It should sign after the original author.

Comment (cybertron) [reply hack/ocp-rebase.sh:51]: > If the commit was not signed off, it should not be merged at all, the github/openshift bot should be smart enough to not merged it and ask the author to sign.

We don't require DCO in openshift though. There are definitely commits in this repo that are not signed off by their original author.

> For the second question: As others open source projects, the person who did the rebase must sign as well to indicate who did the rebase. It should sign after the original author.

It feels weird to me to sign off on a patch if you make no modifications to it and it was already present in the repo. That said, I guess it's not going to hurt anything, so ¯\_(ツ)_/¯

**Comment by cybertron** [comment]
/lgtm

CI wasn't broken by a script change that isn't used at runtime, and we know the ovn job is flaky. No need to spend more ci resources on this.

**Pull request merged**